exo-go/src/service_helpers/index.go

```diff
@@ -27,3 +27,31 @@ func GetExistingServices(services types.Services) []string {
 	}
 	return existingServices
 }
+
+// GetServiceConfigs returns a map that maps each service to its serviceConfig
+func GetServiceConfigs(services types.Services) map[string]types.ServiceConfig {
+	serviceConfigs := make(map[string]types.ServiceConfig)
+	for service, serviceData := range services.Private {
+		serviceConfigs[service] = serviceData
+	}
+	for service, serviceData := range services.Public {
+		serviceConfigs[service] = serviceData
+	}
+	return serviceConfigs
+}
+
+// GetSilencedServices returns a slice of silenced services
```

Comment:
The reader knows that this function returns a slice, so no point in repeating this in the comment. 

// GetSilencedServices returns the services that are configured as silent.